data/test/scenarios/test_add_or_sub_in_leadership.cfg

```diff
@@ -60,55 +78,60 @@
             id=alice
             to_location=2
         [/move_unit]
+
         # Fight
-        [store_unit]
-            [filter]
-                id=bob
-            [/filter]
-            variable=b
-        [/store_unit]
-        [store_unit]
-            [filter]
-                id=alice
-            [/filter]
-            variable=a
-        [/store_unit]
         [do_command]
             [attack]
                 weapon=0
                 defender_weapon=0
                 [source]
-                    x,y=$a.x,$a.y
+                    x,y=12,3
                 [/source]
                 [destination]
-                    x,y=$b.x,$b.y
+                    x,y=13,3
```

Comment:
It seems like to issue is that [move_unit] used `to_location` rather than providing the x and y coordinates.

While I have made some larger changes in order to make the tests easier to understand/write documentation for, I am also trying to avoid refactoring when I can so I can focus just on getting the documentation done.